decent-dao/fractal-interface
# Deploy Azorius

## Description
This PR implements Azorius deployment for existing Multisig safe - either one created through Fractal or any random Safe.


## Notes



## Issue / Notion doc (if applicable)



## Testing
1. Modify governance of existing Multisig DAO or just Safe where you're signer
2. Make sure proposal created properly
3. Execute proposal
4. Verify that DAO now is displayed as Azorius Token Voting DAO with proper gov module settings passed during "Modify Governance" form fill


## Screenshots (if applicable)
